bin/prover-client/src/args.rs

```diff
@@ -117,14 +117,25 @@ pub struct Args {
         description = "wait time in milliseconds for the prover manager loop",
         default = "1_000"
     )]
-    pub loop_interval: u64,
+    pub poll_interval: u64,
```

Comment:
Maybe changing the docstring as well `loop` -> `pooling` (or something like that)